car/README.md

```diff
@@ -0,0 +1,108 @@
+# Certified ARchive
+
+CARs (Certified ARchives) are archives for IPLD DAGs.
+
+## Summary
+
+CARs are archives of IPLD DAGs. They are:
+
+1. Certified
+2. Seekable
+3. Compact
+4. Reproducible
+5. Simple and Stable
+
+The actual format is just a (mostly) recursively-defined topological sort of the
+desired DAG with some metadata for fast traversal.
+
+```
+CID
+len(ROOT)
+ROOT
+[
+  CHILD-1-OFFSET
+  ...
+]
+[
+  len(CHILD-1)
+  CHILD-1
+  [
+    CHILD-1/1-OFFSET
+    ...
+  ]
+  ...
+]
+```
+
+Offsets are relative, offsets for missing children use a sentinel value.
+
+We only bother including the root CID because all the other CIDs are embedded in
+the objects themselves. This saves space and *forces* parsers to actually
+traverse the DAG (hopefully validating it).
```

Comment:
I am not sure I see the benefit of "forcing a parser to traverse the DAG".

Reply:
Good question. For one, it ensures that the CAR is actually one giant DAG rooted at that CID. However, that may not even be worth mentioning (space is, IMO, sufficient).